### PR TITLE
chore: replace eslint-config-ovh by eslint-config-airbnb-base

### PR DESCRIPTION
**Pull request by antleblanc**
### Description of the Change

cd0c235 — chore: replace eslint-config-ovh by eslint-config-airbnb-base

### Benefits

Aligned  with [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript).

ref: SCFRCA-2107

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -140,9 +140,9 @@ module.exports = function (grunt) {
                 }),
                 dest: "."
             },
-            options: {
-                eslintConfig: require("./.eslintrc.js")
-            }
+            // options: {
```
Comment (FredericEspiau) [inline Gruntfile.js]: To remove

File: .eslintrc [hunk 2]
```diff
@@ -1,34 +1,24 @@
...
         "validator": true,
-        "Raven": true,
-        "RFB": true,
-        "WebUtil": true,
-        "RandExp": true,
-        "EprotectIframeClient": true,
         "zE": true
+    },
+    "rules": {
+        "no-bitwise": ["error", { "allow": ["~"] }]
```
Comment (FredericEspiau) [inline .eslintrc]: You can change all `~xxx.indexOf` to `-1 !== xxx.indexOf`, or else we never will get rid of `~`

Comment (blaryjp) [reply .eslintrc]: ~ FTW!

Comment (FredericEspiau) [reply .eslintrc]: You're part of the problem @blaryjp

File: client/app/account/billing/atInternetTracking.config.js [hunk 3]
```diff
@@ -1,43 +1,52 @@
...
-        $rootScope.$on(AUTORENEW_EVENT.TERMINATE, (event, data) => trackServiceClickEvent(event, data));
-        $rootScope.$on(AUTORENEW_EVENT.TERMINATE_AT_EXPIRATION, (event, data) => trackServiceClickEvent(event, data));
-        $rootScope.$on(AUTORENEW_EVENT.CANCEL_TERMINATE, (event, data) => trackServiceClickEvent(event, data));
-        $rootScope.$on(AUTORENEW_EVENT.ACTIVATE_AUTO_PAYMENT, (event, data) => trackServiceClickEvent(event, data));
-        $rootScope.$on(AUTORENEW_EVENT.DEACTIVATE_AUTO_PAYMENT, (event, data) => trackServiceClickEvent(event, data));
-        $rootScope.$on(AUTORENEW_EVENT.MODIFY, (event, data) => trackServiceClickEvent(event, data));
-        $rootScope.$on(AUTORENEW_EVENT.PAY, (event, data) => trackServiceClickEvent(event, data));
+angular.module('Billing.services').run(['$rootScope', 'atInternet', 'AUTORENEW_EVENT', 'PAYMENT_EVENT', 'OVH_ACCOUNT_EVENT', 'FIDELITY_EVENT',
+  function ($rootScope, atInternet, AUTORENEW_EVENT, PAYMENT_EVENT, OVH_ACCOUNT_EVENT,
+    FIDELITY_EVENT) {
```
Comment (JeremyDec) [inline client/app/account/billing/atInternetTracking.config.js]: No alphabetical order ?

Comment (antleblanc) [reply client/app/account/billing/atInternetTracking.config.js]: According to the large amount of files updated in that pull request I didn't sort every parameters. I just let `ESLint` fix all errors encountered with the `--fix` flag.

Of course that would be a great enhancement to sort all parameters according to our guidelines but lets tackle this in smaller and separate tasks, right?

Comment (JeremyDec) [reply client/app/account/billing/atInternetTracking.config.js]: No problem

**Comment by jleveugle** [comment]
Hello @antleblanc,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**